service/src/main/kotlin/app/cash/backfila/dashboard/GetServicesAction.kt

```diff
@@ -34,7 +43,16 @@ class GetServicesAction @Inject constructor(
       val services = queryFactory.newQuery<ServiceQuery>()
           .orderByName()
           .list(session)
-      services.map { it.registry_name }
+      val runningByService = queryFactory.newQuery<BackfillRunQuery>()
+          .state(BackfillState.RUNNING)
+          .list(session)
```

Comment:
no groupby in query factory, but this isn't too bad

Reply:
Yeah this will be fine for now. Can optimize later when it is a problem.